lib/killrails.bash

```diff
@@ -0,0 +1,11 @@
+# Quick function to kill a daemonized Rails server
```

Comment:
Please also add the plugin meta information (the `about-plugin` line, e.g. as seen [here](https://github.com/Bash-it/bash-it/blob/master/plugins/available/autojump.plugin.bash#L1) in lines 1 and 2). This will be used when running `bash-it show plugins` so people know what this plugin does.